polatis.py

```
@@ -490,6 +490,32 @@ def get_patch_table_csv(self, patch_list, filename):
+        data = []
+        for patch in patch_list:
+            inx, outx = patch
+            inp = self.get_inport(inx)
+            outp = self.get_outport(outx)
+            inpower = self.get_port_power(int(inp))
+            outpower = self.get_port_power(int(outp))
+            data.append([inx, "Out", inp, inpower])
+            data.append([outx, "In", outp, outpower])
+        return data
+
+    def get_patch_table_csv(self, patch_list, filename):
```

Ruff (0.12.2)

Redefinition of unused `get_patch_table_csv` from line 461

(F811)